### PR TITLE
soc: nordic_nrf: - MPU temperature sensor default

### PR DESCRIPTION
**Pull request by WilliamGFish**
Create consistency across both nrf51 & nrf52 SOC,  set die temp sensor as default if CONFIG_SENSOR=y

Signed-off-by: William Fish <william.fish@manulytica.com>

### CODE REVIEW

**Review by ioannisg — COMMENTED** [review]
Looks ok to me, thanks for your contribution.
Do you mind fixing the issues with the commit message/sign-line?

File: soc/arm/nordic_nrf/nrf52/Kconfig.defconfig.series [hunk 1]
```diff
@@ -22,4 +22,11 @@ config ARCH_HAS_CUSTOM_BUSY_WAIT
 config SYS_POWER_MANAGEMENT
 	default y
 
+if SENSOR
+
+config TEMP_NRF5
```
Comment (carlescufi) [inline soc/arm/nordic_nrf/nrf52/Kconfig.defconfig.series]: This is shorter and sweeter:

```
config TEMP_NRF5
   default SENSOR
```

and you can get rid of the `if SENSOR`

Comment (WilliamGFish) [reply soc/arm/nordic_nrf/nrf52/Kconfig.defconfig.series]: Done, much cleaner method

**Review by carlescufi — APPROVED** [review]

**Review by galak — APPROVED** [review]

**Comment by aescolar** [comment]
@WilliamGFish Please note that you have moved your own master branch while you where, at the same time, using it for this PR. Which has broken this PR.
Please either create a separate branch in your repository for this PR, and point the PR to it, or revert your master branch to its original content.

**Comment by WilliamGFish** [comment]
> @WilliamGFish Please note that you have moved your own master branch while you where, at the same time, using it for this PR. Which has broken this PR.
> Please either create a separate branch in your repository for this PR, and point the PR to it, or revert your master branch to its original content.

Sorry all, I'm new to GitHub hence i'm inadvertently causing mayhem.

**Pull request closed without merging**